cmd/nfd-master/main.go

```diff
@@ -122,5 +126,24 @@ func initFlags(flagset *flag.FlagSet) *master.Args {
 		"Verify worker node name against the worker's TLS certificate. "+
 			"Only takes effect when TLS authentication has been enabled.")
 
+	normalDeniedNs, wildcardDeniedNs := preProcessDeniedNamespaces(args.DenyLabelNs)
+	args.NormalDenyLabelNs = normalDeniedNs
+	args.WildcardDenyLabelNs = wildcardDeniedNs
```

Comment:
Hmm, I don't think this processing actually does anything as it's inside `initFlags()`. We need to do this after the args are parsed.

I'd suggest to move this processing to `pkg/nfd-master/`. Anyway, don't carry the same data in two (potentially conflicting) ways i.e. args.DenyLabelNs plus args.NormalDenyLabelNs and args.WildcardDenyLabelNs

Reply:
Do you have any suggestions on where to store the data, and parse flags? I think that storing data in separate variables can do the job, and for the processing of the DenyLabelNS, do you see that it's appropriate to do it under `pkg/nfd-master/nfd-master.go`, in the function `NewNfdMaster()`?

Reply:
Yeah, I think that would be the logical place. I.e. pass the "raw" arg via `args.DenyLabelNs` and then store the processed data in separate variables in `nfdMaster` struct for example